modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_mikityuk/tests

```diff
@@ -0,0 +1,63 @@
+[Tests]
+  design = 'ADWallHeatTransferCoefficientMikityukMaterial.md'
+  issues = '#24340'
+  [normal_operation]
+    requirement = "The system shall compute the heat conductance using the Mikityuk correlation for "
+                  "the Nusselt number."
+    type = 'CSVDiff'
+    input = 'mikityuk_test.i'
+    csvdiff = 'mikityuk_test_out.csv'
+  []
+  [abnormal_operation]
+    requirement = "The system shall return a warning message when: "
```

Comment:
...when the Mikityuk correlation is used with:

Reply:
done